packages/ui/src/tracking/index.ts

```diff
@@ -0,0 +1,51 @@
+import { useGtm } from '@gtm-support/vue-gtm';
+
+export const GtmEvents = {
```

Comment:
use array, most of is just key:value object duplication of actionTypes so you can expand on that:
```
export const gtmEvents = [...actionType, 'one more', 'more', 'etc'], as const;
export type GtmEvent = (typeof gtmEvents)[number];
```
then you can simplify the switch as it can fallthrough if you're only collecting type.